manufacturing-ai-suite/wind-turbine-anomaly-detection/docs/user-guide/get-started.md

```diff
@@ -2,13 +2,11 @@
 
 -   **Time to Complete:** 30 minutes
 -   **Programming Language:**  Python 3
+<Do we need this??>>
 
-## Prerequisites
+### Configure Docker
```

Comment:
Let's do additionally call out this as a pre-requisite as there are chances ppl missing this whole or part of this step, mainly for newbies to docker and containerization world